views/partials/scan_form.hbs

```diff
@@ -1,4 +1,6 @@
 <form action="/scan" class="email-scan form-group" method="post" data-no-csrf novalidate>
```

Comment:
Personal curiosity, but now that we're passing hidden `_csrf` fields w `{{csrfToken}}`s, do we still need/want `data-no-csrf` form attributes?

Reply:
Yeah, if you're supplying a CSRF token then you should definitely remove the data-no-csrf attribute :)

Reply:
Ah, I think I remember now ... this attribute is a signal to ZAP? Or another of our scanners?